src/main/java/org/springframework/data/repository/PagingAndSortingRepository.java

```diff
@@ -29,7 +29,7 @@
  * @see Page
  */
 @NoRepositoryBean
-public interface PagingAndSortingRepository<T, ID> extends CrudRepository<T, ID> {
+public interface PagingAndSortingRepository<T, ID> extends Repository<T, ID> {
```

Comment:
We need similar changes for the reactive repositories.

Reply:
Yes, that is understood. 
I'll take care of that once I feel confident I'm heading in the right direction with this.